PennMobile/Home/Cells/Group Invites/GSRGroupInviteCell.swift

```diff
@@ -99,33 +101,34 @@ extension GSRGroupInviteCell {
     
     fileprivate func getGroupNameLabel() -> UILabel {
         let label = UILabel()
-        label.font = UIFont.systemFont(ofSize: 24.0, weight: .regular)
+        label.font = .interiorTitleFont
         label.numberOfLines = 1
         return label
     }
-    
-    fileprivate func getAcceptButton() -> UIButton {
+
+    private func getAcceptButton() -> UIButton {
```

Comment:
So does `private` only show up within the same `extension`?

Reply:
I think so, yes!

Reply:
we should style guide this